helm/helm-release.sh.tpl

```diff
@@ -46,14 +46,14 @@ if [ "$FORCE_HELM_VERSION" == "v2" ] || ( [ "$FORCE_HELM_VERSION" != "v3" ] && [
 
     {HELM_PATH} init -c
 
-    echo "{HELM_PATH} upgrade --install --tiller-namespace {TILLER_NAMESPACE} $HELM_OPTIONS --namespace {NAMESPACE} {VALUES_YAML} {RELEASE_NAME} {CHART_PATH}"
+    echo "{HELM_PATH} upgrade --install --tiller-namespace {TILLER_NAMESPACE} $HELM_OPTIONS --namespace {NAMESPACE} --wait {VALUES_YAML} {RELEASE_NAME} {CHART_PATH}"
```

Comment:
`--create_namesapace` flag is missing here. It's available in helm2 apis?

Reply:
No, `--create_namesapace` it's not available in helm2 api